code/modules/research/xenobiology/crossbreeding/warping

```diff
@@ -0,0 +1,796 @@
+		cooldown = world.time + max_cooldown
+		transmute_resin()
+
+/obj/effect/warped_rune/greenspace/proc/transmute_resin()
+	for(var/obj/item/stack/sheet/mineral/plasma/P in T)
+		X = new(T)
+		P.amount--
+		if(P.amount <= 0)
+			qdel(P)
+
+//list of things you can build with resin sheets.
+GLOBAL_LIST_INIT(resin_recipes, list ( \
+	new/datum/stack_recipe("Resin seed", /obj/structure/alien/weeds/node, 10, one_per_turf = 1, on_floor = 1, time = 100), \
+	new/datum/stack_recipe("Resin Wall", /obj/structure/alien/resin/wall, 5, one_per_turf = 1, on_floor = 1 , time = 50), \
+	new/datum/stack_recipe("Fake hatched egg", /obj/structure/alien/egg/burst, 5, one_per_turf = 1, on_floor = 1), \
+	))
+
+/obj/item/stack/sheet/xeno_resin/get_main_recipes()
+	.= ..()
+	. += GLOB.resin_recipes
+
+
+
+
+/* pink rune, makes people happier when they get hugged on it */
+
+/obj/item/slimecross/warping/pink
+	colour = "pink"
+	effect_desc = "Draw a rune that hugs whoever steps on it.Only useful to friendless people."
+	runepath = /obj/effect/warped_rune/pinkspace
+
+/obj/effect/warped_rune/pinkspace
+	desc = "Love is the only reliable source of happiness we have left.But like everything, it comes with a price."
+	icon_state = "love_rune"
+	var/mob/living/carbon/human/loved
+
+
+/obj/effect/warped_rune/pinkspace/Initialize()
+	.=..()
+	RegisterSignal(T,COMSIG_ATOM_ENTERED,.proc/hug_rune)
+
+///adds the hug mood effect along with the sound effect.
+/obj/effect/warped_rune/pinkspace/proc/hug_rune()
+	for(var/mob/living/carbon/human/H in T)
+		playsound(T, "sound/weapons/thudswoosh.ogg", 50, TRUE)
+		SEND_SIGNAL(H, COMSIG_ADD_MOOD_EVENT, "hug", /datum/mood_event/hug)
+		H.visible_message(H, "<span class='notice'>You feel a magical embrace</span>")
+
+
+
+/*Gold rune : Turn things over it to gold, completely fucks over economy */
+
+/obj/item/slimecross/warping/gold
+	colour = "gold"
+	runepath = /obj/effect/warped_rune/goldspace
+	effect_desc = "draw a rune capable of turning nearly anything put on it to gold."
+
+/obj/effect/warped_rune/goldspace
+	icon_state = "midas_rune"
+	desc = "When everything is abundant,it becomes easy to forget when one had to work to obtain anything."
+
+/obj/effect/warped_rune/goldspace/Initialize()
+	.=..()
+	RegisterSignal(T,COMSIG_ATOM_ENTERED,.proc/midas_touch)
+
+///turn anything on the tile to a golden version of themselves. Sheets of any kind will be directly turned into gold bars.
+/obj/effect/warped_rune/goldspace/proc/midas_touch()
+	for(var/obj/item/stack/sheet/S in T)
+		if(istype(S,/obj/item/stack/sheet/mineral/gold)) //can't turn gold into gold
+			return
+		var/obj/item/stack/sheet/mineral/gold/G = new(T)
+		G.amount = rand(0, S.amount) //will give a random amount of gold between 0 and the amount of the original sheets
+		qdel(S)
+		return
+
+	for(var/obj/item/I in T)
+		I.material_flags = MATERIAL_COLOR | MATERIAL_ADD_PREFIX | MATERIAL_AFFECT_STATISTICS
+		I.set_custom_materials(list(/datum/material/gold=500)) 
+
```

Comment:
honestly, the effect of this gold rune probably should have been the effect of the metal rune. despite their name, gold slimes are associated with summoning mobs, not the gold material. gold is a metal, so it falls under the domain of metal slimes. I was expecting a gold rune to summon demons and shit in a satanic ritual, not turn stuff into gold.

suggested replacement effect: If a unique station pet (not parrots from a parrot crate, holo pets from the holodeck, or shit like that, I mean an ACTUAL unique station pet, like Ian, Poly, Runtime, Citrus, Tom, etc.) dies upon a gold rune (or has its corpse dragged onto the rune), the rune consumes the pet's corpse, creates a netherworld link (https://github.com/tgstation/tgstation/blob/master/code/modules/mob/living/simple_animal/hostile/netherworld.dm) with a nerfed mob cap (say, 5 mobs) on its tile (if there isn't already a netherworld link there), then deletes itself.